mmbiblio.raw.html

```diff
@@ -0,0 +1,160 @@
+      &nbsp;<A HREF="mmset.html">MPE Home</A>&nbsp; &gt;
+      &nbsp;Bibliographic Cross-References
+      </FONT>
+    </TD>
+  </TR>
+</TABLE>
+
+<HR NOSHADE SIZE=1>
+
+<B><FONT COLOR="#006633">Bibliographic
+Cross-References</FONT></B>&nbsp;&nbsp;&nbsp;This table collects in one
+place the bibliographic references made in the Metamath Proof Explorer's
+axiom, definition, and theorem Descriptions.  If you are studying a
+particular set theory book, this list can be handy for finding out where
+any corresponding Metamath theorems might be located.  Keep in mind that
+we usually give only one reference for a theorem that may appear in
+several books, so it can also be useful to browse the Related Theorems
+around a theorem of interest.
+
+<P>
+<!--
+<CENTER><TABLE CELLSPACING=0 CELLPADDING=5
+SUMMARY="Bibliographic Cross-Reference colors"><TR>
+
+<TD>Color key:&nbsp;&nbsp;&nbsp;</TD>
+<TD BGCOLOR="#EEFFFA"><A
+HREF="mmset.html"><IMG SRC="mm.gif" BORDER=0
+ALT="Metamath Proof Explorer" HEIGHT=32 WIDTH=32 ALIGN=MIDDLE>
+&nbsp;Metamath Proof Explorer</A></TD>
+
+<TD WIDTH=10>&nbsp;</TD>
+
+<TD BGCOLOR="#FAEEFF"><A HREF="mmhil.html"><IMG SRC="atomic.gif"
+BORDER=0 ALT="Hilbert Space Explorer" HEIGHT=32 WIDTH=32 ALIGN=MIDDLE>
+&nbsp;Hilbert Space Explorer</A></TD>
+
+</TR></TABLE></CENTER>
+-->
+
+<CENTER>
+<TABLE CELLSPACING=0 CELLPADDING=5
+SUMMARY="Color key"><TR>
+
+<TD>Color key:&nbsp;&nbsp;&nbsp;</TD><TD BGCOLOR="#EEFFFA" NOWRAP><A
+HREF="mmset.html"><IMG SRC="mm.gif" BORDER=0
+ALT="Metamath Proof Explorer" HEIGHT=32 WIDTH=32 ALIGN=MIDDLE>
+&nbsp;Metamath Proof Explorer</A></TD>
+
+<TD WIDTH=10>&nbsp;</TD>
+
+<TD BGCOLOR="#FAEEFF" NOWRAP><A HREF="mmhil.html"><IMG SRC="atomic.gif"
+BORDER=0 ALT="Hilbert Space Explorer" HEIGHT=32 WIDTH=32 ALIGN=MIDDLE>
+&nbsp;Hilbert Space Explorer</A></TD>
+
+<TD WIDTH=10>&nbsp;</TD>
+
+<TD BGCOLOR="#FFFFD9" NOWRAP><A HREF="mmtheorems.html#sandbox:bighdr"><IMG
+SRC="_sandbox.gif" BORDER=0 ALT="User Mathboxes" HEIGHT=32 WIDTH=32
+ALIGN=MIDDLE> &nbsp;User Mathboxes</A></TD>
+
+<TD WIDTH=10>&nbsp;</TD>
+
+</TR></TABLE>
+</CENTER>
+
+
+<!-- <P><CENTER><TABLE BORDER CELLSPACING=0 BGCOLOR="#EEFFFA" F7F7FF  FAEEFF -->
+<P><TABLE BORDER CELLSPACING=0 BGCOLOR="#EEFFFA"
+SUMMARY="Bibliographic Cross-Reference">
+<CAPTION><B>Bibliographic Cross-Reference for the Metamath Proof Explorer</B></CAPTION>
+
+<TR><TH>Bibliographic Reference</TH><TH>Description</TH><TH>Metamath Proof Explorer Page(s)</TH></TR>
+
+<!-- To regenerate the automatically generated content, do the following
+     in the Metamath program:
+       metamath
+       MM> read set.mm
+       MM> write bibliography mmbiblio.html
+-->
+
+<!-- Start of automatically generated section -->
+<!-- do not change the next comment -->
+<!-- #START# -->
+<!-- #END# -->
+<!-- Do not change the preceding comment -->
+<!-- End of automatically generated section -->
+
+</TABLE>
+
+<HR NOSHADE SIZE=-1> <CENTER><I>
+This page was last updated on 1-Jan-2000.
```

Comment:
Maybe you can add a comment that the date on that line is automatically updated (or put that comment with the explanation a few lines above), as you did in the explanation of the PR. Is it better to write a "non-date" instead of 1-Jan-2000 (e.g., dd-Mmm-yyyy) ?

Reply:
I didn't check the granularity of the replacement pass, but if it's using a regex to find this line it might be problematic if it's not a valid date.

Reply:
Ok. What about adding the comment as I proposed above ?

Reply:
```suggestion
<!-- The line below is automatically updated. -->
This page was last updated on 1-Jan-2000.
```